storage/tests/c/flash.c

```diff
@@ -146,3 +146,43 @@ secbool flash_write_word(uint16_t sector, uint32_t offset, uint32_t data) {
   flash[0] = data;
   return sectrue;
 }
+
+secbool flash_write_block(uint16_t sector, uint32_t offset,
+                          const flash_block_t block) {
+#if FLASH_BLOCK_WORDS == 1 || defined FLASH_BIT_ACCESS
+  return flash_write_word(sector, offset, block[0]);
```

Comment:
I don't understand why you assume that a block flash memory with one-word blocks automatically has bit access.

Reply:
I would say the assumption here is the other way around: flash with bit access has block size 1 word. This seems like an acceptable assumption to me, if you can write single bits you can also write single words, no need for blocks to be any larger.

Reply:
> I would say the assumption here is the other way around: flash with bit access has block size 1 word.

This doesn't make sense to me. If `FLASH_BIT_ACCESS` is set, `FLASH_BLOCK_WORDS` is not used, which means that block size doesn't even need to be defined for a flash with bit access.

My complaint actually is that the code doesn't allow you to define a block flash with the block of size of one word. On the other hand, I don't think this is important, because don't have such a flash.

Reply:
ah i think i understand now

https://github.com/trezor/trezor-firmware/commit/540a4c2d133e52f9151e3f5a14624dfe29fb25c2